package.sh

```diff
@@ -375,6 +382,12 @@ if [ $? -ne 0 ]; then
     cleanup_exit 1
 fi
 
+cp $LOGROTATE $TMP_WORK_DIR/$LOGROTATE_DIR/influxdb.conf
```

Comment:
The logrotate config should be at `/etc/logrotate.d/influxdb`, which I think matches this line except I'd leave off the `.conf`.